time-tracker-webapi/test/TimeTracker.Api.Test/HttpClientWithInMemoryDatabase.cs

```diff
@@ -22,6 +22,7 @@ public HttpClientWithInMemoryDatabase()
 
         private void ConfigureWebHost(IWebHostBuilder builder)
         {
+            builder.UseSetting("Environment", "Test");
```

Comment:
Is there a reason it cannot be "Development"?

Reply:
Try changing it to Development and run the tests.... the WAF thingy will fail b/c it will try to instantiate the Static Files with the `node_modules` folder - but that does not exist in test land so it will throw an exception. Had to slightly change startup cs as well